verifier/src/report_body.rs

```diff
@@ -315,6 +316,40 @@ impl<T: Accessor<MrSigner>> Verifier<T> for MrSignerVerifier {
     }
 }
 
+/// Verify the [`ReportData`] is as expected.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct ReportDataVerifier {
+    expected_report_data: ReportData,
+}
+
+impl ReportDataVerifier {
+    /// Create a new [`ReportDataVerifier`] instance.
+    ///
+    /// # Arguments:
+    /// * `expected_report_data` - The expected report data.
```

Comment:
Nit: the comment doesn't add any clarifying details so would recommend removing the parameter callout altogether.

Reply:
Thanks!
Bad habit from document all the things even if when the names match.  I can't find rust official docs, but it does seem rust prefers "clear" argument names over arguments being documented like this suggests https://www.tangramvision.com/blog/making-great-docs-with-rustdoc#-arguments